kyaml/openapi/kubernetesapi/openapiinfo.go

```diff
@@ -6,21 +6,13 @@
 package kubernetesapi
 
 import (
-	"sigs.k8s.io/kustomize/kyaml/openapi/kubernetesapi/v1184"
-	"sigs.k8s.io/kustomize/kyaml/openapi/kubernetesapi/v1186"
-	"sigs.k8s.io/kustomize/kyaml/openapi/kubernetesapi/v1188"
-	"sigs.k8s.io/kustomize/kyaml/openapi/kubernetesapi/v1190"
-	"sigs.k8s.io/kustomize/kyaml/openapi/kubernetesapi/v1191"
+	"sigs.k8s.io/kustomize/kyaml/openapi/kubernetesapi/v1204"
 )
 
-const Info = "{title:Kubernetes,version:v1.18.4}\n{title:Kubernetes,version:v1.18.6}\n{title:Kubernetes,version:v1.18.8}\n{title:Kubernetes,version:v1.19.0}\n{title:Kubernetes,version:v1.19.1}"
+const Info = "{title:Kubernetes,version:v1.19.0}"
```

Comment:
should this be 1.20.4?

Reply:
ah you're right, an error in the script. Will fix, thanks for the catch